src/loaders/svg/tvgSvgLoader.cpp

```diff
@@ -1066,7 +1067,7 @@ static void _handleStrokeDashOffsetAttr(SvgLoaderData* loader, SvgNode* node, co
 static void _handleStrokeWidthAttr(SvgLoaderData* loader, SvgNode* node, const char* value)
 {
     node->style->stroke.flags = (node->style->stroke.flags | SvgStrokeFlags::Width);
-    node->style->stroke.width = _toFloat(loader->svgParse, value, SvgParserLengthType::Horizontal);
+    node->style->stroke.width = _toFloat(loader->svgParse, value, SvgParserLengthType::Width);
```

Comment:
I'd introduce a type "Diagonal" instead of "Width". what do you think?
no matter what the answer will be, I approved :)

Reply:
Diagonal is more appropriate. Thanks!! :) I updated PR.